enterprise/frontend/src/metabase-enterprise/clean_up/CleanupCollectionModal/CleanupCollectionBulkActions.module.css

```diff
@@ -0,0 +1,3 @@
+.container {
+  margin-left: 0 !important;
```

Comment:
```suggestion
  margin-inline-start: 0 !important;
```

Reply:
I'm unsetting the margin-left property that exists on the container component `BulkActionsToast`.

Reply:
nano-nit: at least for me, `unset` would signal this. But I realize there's an argument in favor of `0`